src/main/java/com/redislabs/redisgraph/RedisGraph.java

```diff
@@ -6,15 +6,33 @@
 
 public interface RedisGraph extends Closeable {
 
+    /**
+     * Execute a Cypher query with arguments
+     * @param graphId a graph to perform the query on
+     * @param query Cypher query
+     * @return a result set
+     */
+    ResultSet query(String graphId, String query);
+
     /**
      * Execute a Cypher query with arguments
      * @param graphId a graph to perform the query on
      * @param query Cypher query
      * @param args
      * @return a result set
      */
+    @Deprecated
```

Comment:
please add @deprecated  to the JDoc pointing to the new method

Reply:
done